pkg/daemon/update.go

```diff
@@ -382,6 +389,40 @@ func verifyUserFields(pwdUser ignv2_2types.PasswdUser) error {
 	return nil
 }
 
+func generateKargsCommand(oldConfig, newConfig *mcfgv1.MachineConfig) []string {
```

Comment:
this func is super cool, it could easily be extended to support fips for instance as a last case in the routine itself